docs/glossary.rst

```diff
@@ -58,7 +58,14 @@ Glossary
      The kubelet is the primary "node agent" that runs on each cluster node.
 
      |see K8s docs|
-     `https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/`
+     `kubelet <https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/>`_.
+
+   Namespace
+     A Namespace is a Kubernetes abstraction to support virtual clusters,
```

Comment:
`virtual clusters` is a bit "hard" IMO
You should remove this "virtual cluster" part or do like in the kubernetes docs to specify that it's "multiple virtual clusters backed by the same physical cluster"

Reply:
OK I'll copy paste the "multiple virtual clusters backed by the same physical cluster" from K8s docs ;)